tephra-hbase-compat-1.1-base/src/main/java/org/apache/tephra/hbase/coprocessor/TransactionProcessor.java

```diff
@@ -140,29 +143,37 @@ public void start(CoprocessorEnvironment e) throws IOException {
         ttlByFamily.put(columnDesc.getName(), ttl);
       }
 
-      this.allowEmptyValues = env.getConfiguration().getBoolean(TxConstants.ALLOW_EMPTY_VALUES_KEY,
-                                                                TxConstants.ALLOW_EMPTY_VALUES_DEFAULT);
+      String allowEmptyValuesFromTableDesc = tableDesc.getValue(TxConstants.ALLOW_EMPTY_VALUES_KEY);
+      Configuration conf = getConfiguration(env);
+      boolean allowEmptyValuesFromConfig = (conf != null) ?
+        conf.getBoolean(TxConstants.ALLOW_EMPTY_VALUES_KEY, TxConstants.ALLOW_EMPTY_VALUES_DEFAULT) :
+        TxConstants.ALLOW_EMPTY_VALUES_DEFAULT;
+
+      // If the property is not present in the tableDescriptor, get it from the Configuration
+      this.allowEmptyValues = (allowEmptyValuesFromTableDesc != null) ?
```

Comment:
It might help readability if we move reading `allowEmptyValues` into a private method